lib/internal/repl.js

```diff
@@ -19,11 +20,11 @@ function createRepl(env, opts, cb) {
     cb = opts;
     opts = null;
   }
-  opts = opts || {
+  opts = util._extend({
```

Comment:
+1 If you have problem with implementation fix on master and it can come in another backport